app/preprints/-components/preprint-file-render/template.hbs

```diff
@@ -0,0 +1,50 @@
+<div local-class='file-renderer-container {{if this.isMobile 'mobile'}}'>
+    <div local-class='file-container'>
+        <FileRenderer
+            @allowCommenting={{this.allowCommenting}}
+            @download={{this.primaryFile.links.download}}
+            @version={{this.viewedVersion}}
```

Comment:
It doesn't look like `viewedVersion` is defined on this component. I think we can remove this param